test2/test2/Controllers/CountryModelsController.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.EntityFrameworkCore;
+using test2.Data;
+using test2.Models;
+
+namespace test2.Controllers
+{
+    public class CountryModelsController : Controller
+    {
+        private readonly CountryContext _context;
+
+        public CountryModelsController(CountryContext context)
+        {
+            _context = context;
+        }
+
+        // GET: CountryModels
+        public async Task<IActionResult> Index()
+        {
+            return View(await _context.CountryModel.ToListAsync());
+        }
+
+        // GET: CountryModels/Details/5
+        public async Task<IActionResult> Details(int? id)

```

Comment:
Было бы корректнее с точки зрения построения url http запросов задать шаблон url и перенести переменную пути `id`. Сделать это можно с помощью атрибута над методом как показано ниже. 
Также удобно и понятно для коллег будет обозначить глагол метода при помощи атрибута.
Также добавил предложения по переименованию
```suggestion
        // GET: CountryModels/Details/5
        [HttpGet]
        [Route("countries/{id}/details/")]
        public async Task<IActionResult> Details(int? id)
```
Переделывать не обязательно